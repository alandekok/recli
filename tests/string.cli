str "foo bar baz"
str foo
str "foo
str foo"
str "foo \" bar"
str foo bar
