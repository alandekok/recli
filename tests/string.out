str "foo bar baz"
str foo
str "foo
^ - Invalid input
str foo"
^ - Invalid input
str "foo \" bar"
str foo bar
^ - Invalid input
